code/datums/migrants/migrant_waves/fablefield roles.dm

```diff
@@ -60,7 +59,6 @@
 	show_foreign_examine = TRUE
 
 /datum/outfit/job/roguetown/fablefield/troubadour
-	allowed_patrons = list(/datum/patron/divine/xylix)
```

Comment:
Same matter as above, undocumented; probably should still be kept as a xylix follower.